front/app/containers/ProjectFolderShowPage/index.tsx

```diff
@@ -153,18 +154,35 @@ const ProjectFolderShowPage = memo<{
         ) : projectFolder ? (
           <>
             <StyledContentContainer maxWidth={maxPageWidth}>
-              {userCanEditFolder && (
-                <ButtonBar>
-                  <EditButton
-                    icon="edit"
-                    linkTo={`/admin/projects/folders/${projectFolder.id}/settings`}
-                    buttonStyle="secondary"
-                    padding="5px 8px"
-                  >
-                    <FormattedMessage {...messages.editFolder} />
-                  </EditButton>
-                </ButtonBar>
-              )}
+              <Box display="flex" width="100%">
+                <Box ml="auto" display="flex">
+                  {userCanEditFolder && (
+                    <ButtonBar>
+                      <EditButton
+                        icon="edit"
+                        linkTo={`/admin/projects/folders/${projectFolder.id}/settings`}
+                        buttonStyle="secondary"
+                        padding="5px 8px"
+                      >
+                        <FormattedMessage {...messages.editFolder} />
+                      </EditButton>
+                    </ButtonBar>
+                  )}
+                  <Box ml="8px">
+                    <FollowUnfollow
+                      followableType="project_folders"
+                      followableId={projectFolder.id}
+                      followersCount={projectFolder.attributes.followers_count}
+                      followerId={
+                        projectFolder.relationships.user_follower?.data?.id
+                      }
+                      followableSlug={projectFolder.attributes.slug}
+                      padding="5px 8px"
```

Comment:
Would it be possible to use a multiple of 4 for the padding here? Or move the padding into the parent box maybe? :thinking: